apps/api/src/server/applications/applications.routes.js

```diff
@@ -248,11 +250,49 @@ router.get(
 		}
     */
 	validateApplicationId,
-	validateGetApplicationQuery,
 	validateGetRepresentationsQuery,
 	asyncHandler(getApplicationRepresentations)
 );
 
+router.get(
+	'/:id/representations/:repId',
```

Comment:
I am going to guess that there are going to be more endpoints starting with `/:id/representations` and so I would suggest that we create a folder just for these endpoints and put the controller/endpoints there, following the example of the `/case-team` and other groups of endpoints at the start of this file.

Reply:
i have moved this file into the applications folder.

Reply:
addressed